datageneration/retrieve_combinations.py

```diff
@@ -283,11 +284,11 @@ def check_other_tag_in_properties(self, other_tag: str) -> tuple:
 
         return (exists, -1)
 
-    def request_related_tag_properties(self, tag_key: str, tag_value: str, limit: str = 100) -> List[TagProperty]:
+    def request_related_tag_properties(self, tag_key: str, tag_value: str, limit: int = 100) -> List[TagProperty]:
         combinations = request_tag_combinations(tag_key=tag_key, tag_value=tag_value)['data']
         selected_properties = []
         for combination in combinations:
-            if len(selected_properties) == limit:
+            if len(selected_properties) == limit or combination["together_count"] < self.min_together_count:
```

Comment:
It is not clear for me what togerher_count means, argparser description didn't help me to picture it.

Reply:
Its the number of times two tags occur together. Taginfo calls it "together_count".